pkg/service/backup_handler.go

```diff
@@ -173,21 +232,32 @@ func (h *BackupHandler) runIncrementalBackup(now time.Time) {
 			"name", h.routineName)
 		return
 	}
-	for _, namespace := range h.namespaces {
-		h.runIncrBackupForNamespace(now, namespace)
+	if len(h.incrBackupHandlers) > 0 {
+		slog.Debug("Incremental backup is currently in progress, skipping incremental backup",
+			"name", h.routineName)
+		return
+	}
+
+	client, aerr := aerospike.NewClientWithPolicyAndHost(h.cluster.ASClientPolicy(), h.cluster.ASClientHosts()...)
```

Comment:
Is it possible to reuse the client for full and incremental backups in the same routine?

Reply:
maybe, but its another ticket.
that's a general question, do we want to have client idle connected between backups? I'm not sure.

Reply:
Yes, let's address this in a separate discussion.